common/app/views/fragments/immersiveMainMedia.scala.html

```diff
@@ -0,0 +1,128 @@
+@(page: ContentPage)(implicit request: RequestHeader)
+
+@import layout.ContentWidths.MainMedia
+@import model.{Gallery,Article,ContentPage,GalleryPage}
+@import views.support.TrailCssClasses.toneClass
+@import views.support.ContributorLinks
+
+<div class="content content--immersive tonal tonal--@toneClass(page.item) immersive-main-media
+    @if(page.item.tags.isGallery) { new-gallery }
+    @if(page.item.tags.isArticle) { content--immersive-article }
+    @if(page.item.tags.isUSMinuteSeries) { content--minute-article}
+">
+    @if(!page.item.tags.isGallery) {
+        <div class="gs-container immersive-main-media__logo">
+            <a href="@LinkTo{/}">
+                <span class="u-h">The Guardian</span>
+                @fragments.inlineSvg("guardian-logo-160", "logo", Seq("immersive-main-media__logo__svg"))
+            </a>
+            @page.item match {
+                case article: Article => {
+                    @if(article.isUSMinute) {
+                        <a class="logo--us-election" href="@LinkTo{/us-news/us-elections-2016}">
+                            <span class="u-h">The US Election 2016</span>
+                            @fragments.inlineSvg("us-election-logo", "logo")
+                        </a>
+                    }
+                }
+                case _ => {}
+            }
+        </div>
+    }
+
+    @if(page.item.tags.isArticle && (page.item.elements.hasMainEmbed || page.item.elements.elements("main").isEmpty)) {
+        <div class="immersive-main-media__media">
+            <div class="immersive-main-media__media__loading">
+                <div class="immersive-main-media__media__loading__animation is-updating"></div>
+                <span class="u-h">Loading header</span>
+            </div>
+            @Html(page.item.fields.main)
+        </div>
+    } else {
+        @page.item.elements.mainPicture.map(_.images).orElse(page.item.trail.trailPicture).map { picture =>
+            @fragments.image(
+                picture = picture,
+                classes = Seq("immersive-main-media__media"),
+                widths = MainMedia.immersive,
+                imageAltText = "",
```

Comment:
indeed, actually it does have alt text - the alt text is an empty string.  This is the correct way to do it for accessiblity so screen readers etc know there's nothing significant about the image.  Without alt text it has to find some other way to describe it, just in case it was significant.